server.js

```diff
@@ -1,3 +1,9 @@
+/*
```

Comment:
okay let's review this after the standup because I think there should be a way for us to give credit where it is due bot not give credit to others on our work.

Reply:
Okay sounds good Eduard. Talk after the meeting.